docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -93,6 +93,48 @@ include-tagged::{sql-specs}/docs.csv-spec[dtIntervalMul]
 
 beta[]
 
+[[sql-functions-current-date]]
+==== `CURRENT_TIMESTAMP/CURDATE`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+CURRENT_DATE
+CURRENT_DATE()
+--------------------------------------------------
+
+*Input*: _none_
+
+*Output*: date
+
+.Description:
+
+Returns the date (no time part) when the current query reached the server.
+It can be used both as a keyword: `CURRENT_DATE` or as a function with no arguments: `CURRENT_DATE()`.
+
+[NOTE]
+Unlike CURRENT_DATE, `CURDATE()` can only be used as a function with no arguments and not as a keyword.
+
+This method always returns the same value within a query.
```

Comment:
How about `This method always returns the same value within a query, no matter how many times it is used inside the query.` to make it more clear?